src/test/java/io/github/ryotan/code/example/Gender.java

```diff
@@ -0,0 +1,49 @@
+/*
+ * These codes are licensed under CC0.
+ *
+ * http://creativecommons.org/publicdomain/zero/1.0/deed
+ * http://creativecommons.org/publicdomain/zero/1.0/deed.ja
+ */
+
+package io.github.ryotan.code.example;
+
+import java.util.function.Predicate;
+
+import io.github.ryotan.code.CodeEnum;
+
+public enum Gender implements CodeEnum<Gender> {
+    NOT_APPLICABLE("9", "適用不能", "not applicable"),
+
+    @ShortLabel("女")
+    FEMALE("2", "女性", "female"),
+
+    @ShortLabel("男")
+    MALE("1", "男性", "male"),
+
+    NOT_KNOWN("0", "不明", "not known");
+
+    private final String value;
+    private final String label;
+
+    @AliasLabel
+    public final String ENGLISH_LABEL;
+
+    @Filter
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Static variable definition in wrong order.](https://www.codacy.com/app/ryotan/code-enum-support/file/3194451349/issues/source?bid=3375378&fileBranchId=3375797#l31)